src/che-workspace-project-manager.ts

```diff
@@ -32,6 +32,9 @@ export class CheWorkspaceProjectManager {
         }
 
         const cloneCommandList = await this.selectProjectToCloneCommands(workspaceConfig);
+        if (cloneCommandList.length <= 0) {
+            return;

```

Comment:
On which case can it be < 0 ? Why not === 0

Reply:
we don't know ...

Reply:
? the function is implemented in the current class, it can never return a negative value

Reply:
Technically array.length is a number and can be a negative value. it depends on the implementation of the JS runtime not ? but i don't mind :)
I have update.

Reply:
> Every Array object has a length property whose value is always a nonnegative integer less than 232.
http://www.ecma-international.org/ecma-262/5.1/index.html#sec-15.4

Reply:
yes this is the spec